packages/builder/src/builder.js

```diff
@@ -61,7 +60,7 @@ export default class Builder {
       })
 
       // Close hook
-      this.nuxt.hook('close', () => this.close())
+      this.nuxt.hook('close', this.close)
```

Comment:
This may lose the context for `close`

Reply:
Damn, right! I wasn't sure about that change and remembered that we talked about that before.